src/package-request.js

```diff
@@ -351,7 +352,11 @@ export default class PackageRequest {
     // prevents us from having to query the metadata for all packages.
     if (filterByPatterns && filterByPatterns.length) {
       const filterByNames = filterByPatterns.map(pattern => normalizePattern(pattern).name);
-      depReqPatterns = depReqPatterns.filter(dep => filterByNames.indexOf(normalizePattern(dep.pattern).name) >= 0);
+      depReqPatterns = depReqPatterns.filter(
+        dep =>
+          filterByNames.indexOf(normalizePattern(dep.pattern).name) >= 0 ||
+          (flags && flags.pattern && normalizePattern(dep.pattern).name.includes(flags.pattern)),
```

Comment:
Shouldn't you use `micromatch` rather than `includes`, which will only search for the substring?

Reply:
Yeah you're right. Changed now.